src/modules/canvas-renderer/canvas-renderer.ts

```diff
@@ -457,17 +466,31 @@ export class CanvasRenderer implements Renderer {
               );
             }
           } else {
-            this.ctx.drawImage(
-              canvasToPaint,
-              0, // paint.display.points[index * 5 + 1],
-              0, // paint.display.points[index * 5 + 2],
-              paint.display.points[index * 5 + 3] - paint.display.points[index * 5 + 1],
-              paint.display.points[index * 5 + 4] - paint.display.points[index * 5 + 2],
-              x,
-              y,
-              width + Number.MIN_VALUE,
-              height + Number.MIN_VALUE
-            );
+            if (isFirefox) {
+              this.ctx.drawImage(
+                canvasToPaint,
+                0, // paint.display.points[index * 5 + 1],
+                0, // paint.display.points[index * 5 + 2],
+                paint.display.points[index * 5 + 3] - paint.display.points[index * 5 + 1],
+                paint.display.points[index * 5 + 4] - paint.display.points[index * 5 + 2],
+                x,
+                y,
+                width + 1,
+                height + 1
+              );
+            } else {
+              this.ctx.drawImage(
+                canvasToPaint,
+                0, // paint.display.points[index * 5 + 1],
+                0, // paint.display.points[index * 5 + 2],
+                paint.display.points[index * 5 + 3] - paint.display.points[index * 5 + 1],
+                paint.display.points[index * 5 + 4] - paint.display.points[index * 5 + 2],
+                x,
+                y,
+                width + Number.MIN_VALUE,
+                height + Number.MIN_VALUE
```

Comment:
@stephenwf should this be the one that has the addition?
```suggestion
                width + Number.MIN_VALUE + .5,
                height + Number.MIN_VALUE + .5
```

Reply:
Ah good catch! Thanks. Pushed that up